### PR TITLE
Add I2C option for Lidar Lite v3

### PR DESCRIPTION
**Pull request by mzahana**
This pull request adds the option of starting Lidar Lite on I2C

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/rcS [hunk 1]
```diff
@@ -746,6 +746,17 @@ then
...
 	fi
 
+	# Lidar-Lite on I2C
+	if param compare SENS_EN_LL40LS 2
+	then
+		if pwm_input start
+		then
+			if ll40ls start i2c
+			then
+			fi
```
Comment (mhkabir) [inline ROMFS/px4fmu_common/init.d/rcS]: Remove the pwm_input block. We don't need it for i2c.

Comment (mzahana) [inline ROMFS/px4fmu_common/init.d/rcS]: Sorry. Forgot to remove it. Will do now.

**Comment by mzahana** [comment]
Tested  and works fine. I can see reasonable distance on QGC when ```SENS_EN_LL40LS=2```.

**Comment by mzahana** [comment]
Can this be added to coming stable release?

**Comment by mhkabir** [comment]
Will be in release v1.6, thanks a lot!

**Comment by gjkunde** [comment]
How would I test this on AERO RTF and PIXHAWK2.1 ? By
just wiring the Lidar Lite V3 into the I2C bus and then set
SENS_EN_LL40LS=2 and look at the analyze widget or is
there a more direct way ?

**Pull request merged**